README.md

```diff
@@ -220,6 +220,18 @@ It is also possible to cache gems manually, but this is not recommended because
 There are many concerns which means using `actions/cache` is never enough for caching gems (e.g., incomplete cache key, cleaning old gems when restoring from another key, correctly hashing the lockfile if not checked in, OS versions, ABI compatibility for `ruby-head`, etc).
 So, please use `bundler-cache: true` instead and report any issue.
 
+### ASan Debugging
+
+Ruby can be built with AddressSanitizer (ASan) for debugging memory issues. You can use the `asan` variant of Ruby for this:
```

Comment:
Could you move it around lines 23? That's where we explain other similar variants.
There is no need for a YAML snippet, it should be obvious.
It's also ubuntu-24.04 only, we should mention that.

Reply:
Mmh, maybe we need its own section to document env vars and stuff, see https://github.com/ruby/setup-ruby/pull/653#issuecomment-2404707671
But anyway I think we should mention `asan` near line 23 and that it's ubuntu-24.04 only

Reply:
Yes, think it needs it's own section, but we could also add a note on line 23.